api/src/main/scala/org/alephium/api/model/BuildTransaction.scala

```diff
@@ -18,15 +18,16 @@ package org.alephium.api.model
 
 import org.alephium.protocol.PublicKey
 import org.alephium.protocol.model.{Address, NetworkType}
-import org.alephium.protocol.vm.LockupScript
+import org.alephium.protocol.vm.{GasPrice, LockupScript}
 import org.alephium.util.{TimeStamp, U256}
 
 @SuppressWarnings(Array("org.wartremover.warts.DefaultArguments"))
 final case class BuildTransaction(
     fromKey: PublicKey,
     toAddress: Address,
     lockTime: Option[TimeStamp] = None,
-    value: U256
+    value: U256,
+    gasPrice: Option[GasPrice] = None
```

Comment:
@tdroxler Why I don't need to update examples and openapi.json for this change ?

Reply:
for the examples, the construction is still valid, as you puted a default value. But it's strange for the openapi.json, I would need to check.

Reply:
Got it, `BuildTransaction` is used in the `ServerUtils`, but isn't used in the API itself, we are using query params (as it's a `GET`) https://github.com/alephium/alephium/blob/71dbe3a1451a68ce62c4fa03398047337ca4cc32/api/src/main/scala/org/alephium/api/Endpoints.scala#L196-L205   
   
Which means you also need to add the query param

Reply:
@tdroxler Why don't we use a json as input like Wallet ?